node/full_node_integration_test.go

```diff
@@ -47,12 +45,11 @@ func TestAggregatorMode(t *testing.T) {
 	key, _, _ := crypto.GenerateEd25519Key(rand.Reader)
 	genesisValidators, signingKey := types.GetGenesisValidatorSetWithSigner()
 	blockManagerConfig := config.BlockManagerConfig{
-		BlockTime:   1 * time.Second,
-		NamespaceID: types.NamespaceID{1, 2, 3, 4, 5, 6, 7, 8},
+		BlockTime: 1 * time.Second,
```

Comment:
[non-blocking] this is set to 1 second in a few places. Should it be a constant?